### PR TITLE
[Chore](profile) add some profile on ReaderInit

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
### What problem does this PR solve?
add some profile on ReaderInit
Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by gavinchou — DISMISSED** [review]

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/olap/rowset/beta_rowset_reader.cpp [hunk 1]
```diff
@@ -253,6 +255,7 @@ Status BetaRowsetReader::get_segment_iterators(RowsetReaderContext* read_context
     const bool use_lazy_init_iterators =
             !is_merge_iterator && _read_context->reader_type == ReaderType::READER_QUERY;
```
Comment (yiguolei) [inline be/src/olap/rowset/beta_rowset_reader.cpp:256]: 给230 行的load_segments 也加一个timer

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/olap/rowset/beta_rowset_reader.cpp [hunk 2]
```diff
@@ -253,6 +258,7 @@ Status BetaRowsetReader::get_segment_iterators(RowsetReaderContext* read_context
...
     for (int i = seg_start; i < seg_end; i++) {
+        SCOPED_RAW_TIMER(&_stats->rowset_reader_create_iterators_timer_ns);
```
Comment (yiguolei) [inline be/src/olap/rowset/beta_rowset_reader.cpp:261]: 这个是不是移动到for 循环外，否则for 循环多了，这个是不是影响还是很大的？

**Review by yiguolei — APPROVED** [review]

**Review by Mryange — APPROVED** [review]

**Pull request merged**